validation/utils_test.go

```diff
@@ -51,6 +51,12 @@ func TestAreCloseInts(t *testing.T) {
 
 func SkipCheckIfFrontierChain(t *testing.T, chain superchain.ChainConfig) {
 	if chain.Type == superchain.Frontier {
-		t.Skip()
+		t.Skip("Skipping check (Frontier chain)")
+	}
+}
+
+func SkipCheckIfBlurred(t *testing.T, chain superchain.ChainConfig) {
+	if focus && focussedChainId != chain.ChainID {
```

Comment:
Can we remove the `focus` bool to simplify the UX to:
```
go test ./... -v -chain-id=<your-chain-id-here>
```

And this check then becomes `if focussedChainId != chain.ChainID {`

Reply:
[simplify API for focussing tests](https://github.com/ethereum-optimism/superchain-registry/pull/154/commits/35669302c1b781efed6ffe63133cbe94ce643912)